source/_integrations/spotify.markdown

```diff
@@ -31,8 +31,8 @@ To create the required Spotify application:
 - Select **Create An App**. Enter any name and description.
 - Once your application is created, view it and copy your **Client ID** and **Client Secret**, which are used in the Home Assistant configuration file.
 - Add a **Redirect URI** in one of the following forms:
-  - If you are not using SSL: `http://<your_home_assistant_url_or_local_ip>/auth/external/callback`
-  - If you are using SSL: `https://<your_home_assistant_url_or_local_ip>/auth/external/callback`
+  - If you are not using SSL: `http://<your_home_assistant_url_or_local_ip>:8123/auth/external/callback`
+  - If you are using SSL: `https://<your_home_assistant_url_or_local_ip>:8123/auth/external/callback`
```

Comment:
That depends on your setup...
Mine doesn't run on 8123...

Reply:
Good point. Am new to HA.
So what would be the most suitable way to indicate that a port is most likely required, as the instructions were unclear?

Reply:
Personally, I don't think it is needed, as the template says it needs your URL (of which the port is part of).

However, is we do this, lets make the port number a placeholder as well..

e.g., `http://<your_home_assistant_url_or_local_ip>:<port>/auth/external/callback`

Reply:
Sounds good - want to make the change or should I ?